manifests/s/SpencerSoft/Chess/1.0.0/SpencerSoft.Chess.installer.yaml

```diff
@@ -7,8 +7,8 @@ Installers:
 - InstallerLocale: en-US
   Architecture: x86
   InstallerType: msi
-  InstallerUrl: https://github.com/spencerparkin/ChessEngine/releases/download/v1.2/ChessInstaller.msi
-  InstallerSha256: A9183F4E5F4A4AB104A1182B248768AD1E598C92D61AC084F635FB3800273E32
+  InstallerUrl: https://github.com/spencerparkin/ChessEngine/releases/download/v1.3/ChessInstaller.msi
```

Comment:
Shouldn't the PackageVersion be updated from `1.0.0` to `1.3` to match the URL version? This would allow upgrading to work correctly - performed this way, I believe the WinGet client will see these as the same version, so users who already have it installed wouldn't see an upgrade from `WinGet upgrade`. 

(Please note that updating the PackageVersion has to be performed in all 3(?) manifest files, and their paths have to be updated to match as well.)